portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsTemplate.testcase

```diff
@@ -216,4 +216,49 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test UsePublicationNameSetInSelectedPublicationTemplate {
+		PublicationsTemplate.gotoPublicationTemplate();
+
+		PublicationsTemplate.addPublicationTemplate(
+			publicationDescription = "Publications Description ${CURRENT_USERNAME} 1",
+			publicationName = "Publication Name ${CURRENT_USERNAME} 1",
+			templateDescription = "Template Description 1",
+			templateName = "Template Name 1");
+
```

Comment:
Hi @CaiocSiqueira, thanks for working on the ticket. I made a small change before adding another publication template, ususlly we need to go to template agian then adding another one to avoid some falky issues.

cc @xbrianlee